src/features/movieList/movieListSaga.js

```diff
@@ -0,0 +1,39 @@
+import { delay, call, put, takeLatest } from "redux-saga/effects";
+import { getGenres, getMovies } from "./movieListAPI";
+import {
+  fetchMovies,
+  fetchMoviesError,
+  fetchMoviesSuccess,
+  fetchGenres,
+  fetchGenresError,
+  fetchGenresSuccess,
+} from "./movieListSlice";
+
+const loadingDelay = 2000;
+
+function* fetchMoviesHandler() {
+  try {
+    yield delay(loadingDelay); //for loader demo purpose
```

Comment:
Pozostawiony komentarz, nie wiem czy zapomniałeś, czy na tą chwilę jest tu potrzebny :D

Reply:
Krzysiek proponował zostawić taki komentarz, jeśli delay jest użyty tylko do pokazania loadera. Jeśli Cię razi to możemy usunąć ^^

Reply:
Niech zostanie :D